web/html/src/.eslintignore

```diff
@@ -4,4 +4,44 @@ build/**/*
 build.js
 flow-typed/**/*
 global.d.ts
-imports.d.ts
\ No newline at end of file
+imports.d.ts
+
+# These files need Flow to TS migration before they're lintable again
```

Comment:
Do we need to declare them one by one, instead of grouping and migrating folder by folder?

Reply:
We could ignore whole folders, but that would be too broad. For example, we don't have issues with everything in `components/input`, only `InputBase.js`. Same for other folders. I'd prefer to disable linting for as few files as possible, plus this cuts the list of must-migrate-asap files down considerably.

Reply:
Fine by me then :wink: